src/test/java/com/google/api/codegen/protoannotations/testdata/nodejs_library.baseline

```diff
@@ -1425,6 +1425,35 @@ const argv = require(`yargs`)
   .argv;
 
 sampleTestOptionalRequiredFlatteningParams(argv.param_float, argv.param_long);
+============== file: samples/v1/test_long_running_start_then_cancel_calling_form.js ==============
+// DO NOT EDIT! This is a generated sample ("LongRunningStartThenCancel",  "test_long_running_start_then_cancel_calling_form")
+'use strict';
+
+// sample-metadata:
+//   title: kick off, retrieve name, and cancel
+//   description: Kick off the LRO and cancel it.
+//   usage: node samples/v1/test_long_running_start_then_cancel_calling_form.js
+
+// [START test_long_running_start_then_cancel_calling_form]
+
+const library = require('@google-cloud/library').v1;
+
+/** Kick off the LRO and cancel it. */
+async function sampleGetBigBook() {
+  const client = new library.LibraryServiceClient();
+  const formattedName = client.bookPath('[BOOK_SHELF]', '[BOOK]');
+  const [operation] = await client.getBigBook({name: formattedName});
+
+  // You'll want to save this for later to retrieve your completed operation.
+  console.log(`Operation Name: ${operation.name}`);
+  // Cancel the operation to avoid charges when testing.
+  await client.operationsClient.cancelOperation({name: operation.name});
```

Comment:
Looks like what I would guess! I haven't tried it yet, I'll give it a shot.

/cc @bcoe @jkwlui in case they know right away _(I can't find any examples online)_